src/client-configuration.ts

```diff
@@ -120,6 +120,11 @@ export interface ClientConfiguration {
    */
   typecheck?: boolean;
 
+  /**
+   * Enable or disable performance hints. If no value is provided, performance hints will be disabled.
+   */
+  performanceHints?: boolean;
```

Comment:
As much as this pains me, for better or worse, these types have been using `snake_case` for configuration that gets passed as headers. We should keep these consistent and make this `performance_hints`.

We can move everything user-facing to camel in a future major version change. Until then, it's better to be consistent even if it's sort of silly.

Reply:
o thats cool! I just didn't realize that and I just trend towards the camel case by default